### DISCUSSION TITLE
How can I  programmatically obtain the NU errors

### DISCUSSION THREAD

**Discussion by Smartisa** [NuGet/Home#11938] [Q&A]
I can check the "project assets. json" file to get the log, but how can I  programmatically obtain the NU errors?
![image](https://user-images.githubusercontent.com/47079990/177030593-99a9fde4-ff16-49ca-a80a-0a988613df04.png)

**Comment by erdembayar** [comment]
One way could be just deserializing the json and look for it. I believe you can use https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFile.cs for it.
Let me know if it works for you.

**Reply by Smartisa** [reply]
Is there any other way to get Nu errors by calling the interface instead of relying on this file?
